recipes/elk/webview.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (Ferdium) => {
+    Ferdium.loop(() => {
+        // FIXME: Don't assume 0th element exists
```

Comment:
Please fix this with some defensive programming.

Reply:
I fixed this by testing that the array is not empty (length === 0), is this enough?

Reply:
you have a `FIXME` comment. If you have identified any scenario where this can go wrong, please fix that, else remove the comment.